package.json

```diff
@@ -1,5 +1,7 @@
 {
   "scripts": {
+    "profile" : "node --no-warnings ./scripts/profile.js",
```

Comment:
Please add info on how to use these to the README

Reply:
Sorry, added now.